packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -101,6 +102,8 @@ export function createAutocompleteDom<TItem extends BaseItem>({
     getInputProps: propGetters.getInputProps,
     getInputPropsCore: autocomplete.getInputProps,
     autocompleteScopeApi,
+    // In detached mode we don't want to close the panel when hittin `Tab`.
+    onDetachedTab: isDetached ? noop : undefined,
```

Comment:
We could just forward `isDetached` here to trigger the custom logic?

Reply:
What do you have in mind?